$NetBSD: patch-configure,v 1.3 2012/05/14 05:01:24 sbd Exp $

Change the the cast on gettext() when testing for it so the gcc doesn't
complain with "error: cast from 'char*' to 'int' loses precision",
which causes the test to fail and NLS to be disabled.

--- configure.orig	2012-04-29 05:43:20.000000000 +0000
+++ configure
@@ -17725,7 +17725,7 @@ int
 main ()
 {
 bindtextdomain ("", "");
-return (int) gettext ("") + _nl_msg_cat_cntr + *_nl_domain_bindings
+return * gettext ("") + _nl_msg_cat_cntr + *_nl_domain_bindings
   ;
   return 0;
 }
@@ -18247,7 +18247,7 @@ int
 main ()
 {
 bindtextdomain ("", "");
-return (int) gettext ("") + _nl_msg_cat_cntr + *_nl_expand_alias (0)
+return * gettext ("") + _nl_msg_cat_cntr + *_nl_expand_alias (0)
   ;
   return 0;
 }
@@ -18274,7 +18274,7 @@ int
 main ()
 {
 bindtextdomain ("", "");
-return (int) gettext ("") + _nl_msg_cat_cntr + *_nl_expand_alias (0)
+return * gettext ("") + _nl_msg_cat_cntr + *_nl_expand_alias (0)
   ;
   return 0;
 }
